packages/rebber-plugins/src/type/appendix.js

```diff
@@ -0,0 +1,20 @@
+/* Dependencies. */
+const all = require('rebber/dist/all')
+const clone = require('clone')
+
+/* Expose. */
+module.exports = appendixPlugin
+
+function definitionMacro (ctx, identifier, url) {
+  return `\\label{${identifier}}${url}`
+}
+
+function appendixPlugin (ctx, node) {
+  if (node.children.length === 0) {
+    return ''
+  }
+  const overriddenCtx = clone(ctx)
+  overriddenCtx.definition = definitionMacro
+  const innerText = all(overriddenCtx, node)
+  return `\\begin{appendices}\n${innerText}\n\\end{appendices}`
```

Comment:
Please do:

```diff
-  return `\\begin{appendices}\n${innerText}\n\\end{appendices}`
+  return `\\begin{appendices}\n${innerText.slice(0,-2)}\n\\end{appendices}`
```

To prevent newlines at the end of the appendices.

Reply:
Sounds a bit dangerous if you only want to trim, use `String.prototype.trimEnd()` instead?

Reply:
I like it; updated my comment.

Reply:
updated, thanks for the reviex